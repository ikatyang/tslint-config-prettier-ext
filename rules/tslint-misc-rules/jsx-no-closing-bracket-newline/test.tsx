const fails = (
  <div>
      <span className={ value }
      />
      <a className="asdf"
          href="/foo"
          id="123" /
      >
      <div
          className="qwer"
          id="asdf"
      >
          text
      </div>
  </div>
);
