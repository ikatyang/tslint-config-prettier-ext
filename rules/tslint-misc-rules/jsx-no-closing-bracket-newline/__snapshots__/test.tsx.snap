// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (jsx-no-closing-bracket-newline) /rules/tslint-misc-rules/jsx-no-closing-bracket-newline/test.tsx[4, 7]: closing brackets for jsx elements should not be on newlines
ERROR: (jsx-no-closing-bracket-newline) /rules/tslint-misc-rules/jsx-no-closing-bracket-newline/test.tsx[8, 7]: closing brackets for jsx elements should not be on newlines
ERROR: (jsx-no-closing-bracket-newline) /rules/tslint-misc-rules/jsx-no-closing-bracket-newline/test.tsx[12, 7]: closing brackets for jsx elements should not be on newlines

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const fails = (
  <div>
      <span className={ value }
      />
      <a className=\\"asdf\\"
          href=\\"/foo\\"
          id=\\"123\\" /
      >
      <div
          className=\\"qwer\\"
          id=\\"asdf\\"
      >
          text
      </div>
  </div>
);

======
const fails = (
  <div>
    <span className={value} />
    <a className=\\"asdf\\" href=\\"/foo\\" id=\\"123\\" />
    <div className=\\"qwer\\" id=\\"asdf\\">
      text
    </div>
  </div>
);

>>>>>> after
"
`;
